src/toolkits/activity_classification/activity_classifier.hpp

```diff
@@ -201,7 +201,7 @@ class EXPORT activity_classifier: public ml_model_base {
       const;
 
   // Returns the initial neural network to train
-  virtual std::unique_ptr<neural_net::model_spec> init_model() const;
+  virtual std::unique_ptr<neural_net::model_spec> init_model(bool use_zero_init = false) const;
```

Comment:
We shouldn't use default arguments for virtual functions, due to nonintuitive semantics. See also https://google.github.io/styleguide/cppguide.html#Default_Arguments